migrations/versions/a5a40d843415_changed_sensitive.py

```diff
@@ -0,0 +1,47 @@
+"""changed_sensitive
+
+Revision ID: a5a40d843415
+Revises: d117e6299dc9
+Create Date: 2022-02-19 10:32:31.626236
+
+"""
+from alembic import op
+import sqlalchemy as sa
+from sqlalchemy.dialects import mysql
+
+# revision identifiers, used by Alembic.
+revision = "a5a40d843415"
+down_revision = "d117e6299dc9"
+branch_labels = None
+depends_on = None
+
+
+def upgrade():
+    # ### commands auto generated by Alembic - please adjust! ###
+    op.add_column("projects", sa.Column("non_sensitive", sa.Boolean(), nullable=True))
```

Comment:
Shouldn't the value be converted from the old one to the new one, rather than be dropped completely?

Reply:
So... how do I do that?